app/sender.py

```diff
@@ -75,7 +81,11 @@ async def start(
                     last = await send(buffer)
 
             new_cps = await cps(who)
-            # TODO: add 1 coin to `who` here
+            profile = await client.database.get_profile(guild, who)
+            new_coins = profile.coins + 1
+            await client.database.update_profile(
+                guild, who, UserProfile(coins=new_coins, cps=profile.cps, priority=profile.priority)
+            )
```

Comment:
This is going to conflict a lot. Do you think it's worth it to remove this logic and re-add it once https://github.com/A5rocks/code-jam-11/pull/52 is merged?

Reply:
Yeah, I noticed that when #52 was made. I think it'll make it easier if this was added later. Should it just be commented out for now?

Reply:
Yeah, or even removed straight out because whatever this will become will be different: probably another async function passed in like `cps`.